### DISCUSSION TITLE
Change the cookie icon?

### DISCUSSION THREAD

**Discussion by fgarcia-zorraquino** [dargmuesli/nuxt-cookie-control#180] [Q&A] [answered]
How can i change the cookie icon (svg), or if is possible change it for a text?

Thanks in advance.

**Comment by dargmuesli** [comment]
Would you please specify which icon you're mentioning exactly?

**Comment by fgarcia-zorraquino** [comment]
The cookie icon in the corner of the screen when you close the panel 
![Captura de Pantalla 2024-01-11 a las 10 45 00](https://github.com/dargmuesli/nuxt-cookie-control/assets/153506389/4cfc696a-9b8f-4ee5-9cc3-92140f722cb5)

**Reply by dargmuesli** [reply]
That's currently not possible, but a PR is welcome!

**Reply by dargmuesli** [reply, marked as answer]
It's now possible: https://github.com/dargmuesli/nuxt-cookie-control/pull/265
